### ISSUE TITLE
Doesn't work for 2000 score calculation

### ISSUE THREAD

**Issue by boyiguo1** [boyiguo1/Tutorial-Residential_Segregation_Score#7]
As we are hard coded the variable codes (of 2010) for total, majority, and minority in `prep_tidycensus_data`. It will induce problems when calculating for other years whose variable code changes, e.g. 2000.

**Comment by boyiguo1** [comment]
TODO:

- [x] Fix the bug in the `master` branch
        * Fixed via 6c9737d
- [x] Copy over the fix to Meds_desert page
      * Fixed via 2c8803d

**Comment by boyiguo1** [comment]
Test case:
modify the following configurations.
```{r}
  tar_target(year, 2000),
  tar_target(state, "AL"),
  tar_target(top_lvl, "county"),
  tar_target(btm_lvl, "tract"),
  tar_target(census_code_total, "P003001"), # 2000 Total
  tar_target(census_code_maj,   "P003003"), # 2000 Total White (Majority)
  tar_target(census_code_min,   "P003004"), # 2000 Total Black (Minority)
```
